### DISCUSSION TITLE
Linux install guide simple don't work

### DISCUSSION THREAD

**Discussion by FelipeCODX** [oobabooga/text-generation-webui#1067] [Q&A]
I'm not able to run gpt4-x-alpaca-13b-native-4bit-128g, it's missing a ton of requirements

**Comment by MartinsRepo** [comment]
I made some suggestions above in the discussion, but finally i didn't get running the ubuntu version. The support for the Linux community is very bad here.

**Comment by charder1** [comment]
Been trying to get it to run a model for a couple hours now with no luck either (fedora37)
Same story on windows but I gave up much quicker.

**Comment by csharpner** [comment]
I ran ./start_linux and it installed a lot of stuff and it appeared to download the model I chose, but it looks like it died in "Loading checkpoint shards"?

Hardware:
- Ryzen 7 8 core CPU,
- 32GB DDR4 RAM
- RTX 3090 w/24GB of video RAM
- RTX 3080 Ti
- Plenty of drive space

```shell
Gradio HTTP request redirected to localhost :)
bin /home/owner/Projects/AI/oobabooga_linux/installer_files/env/lib/python3.10/site-packages/bitsandbytes/libbitsandbytes_cuda117.so
Loading EleutherAI_pythia-6.9b-deduped...
Auto-assiging --gpu-memory 23 for your GPU to try to prevent out-of-memory errors.
You can manually set other values.
Loading checkpoint shards:   0%|                                                                                                                                                                                                                                   | 0/2 [00:00<?, ?it/s]Killed

Done!
```
Notice the "killed" on the far right of the results above.

The only instructions say to download the zip and double-click the start icon.  No such thing on the linux download.  I did run the start_linux.sh, but there are no instructions on what to do now.  I don't even know if it successfully installed.  I tried running:

```shell
python webui.py
```

And it says Python isn't installed.

There's just no information available.

**Reply by nousernamesleftmike** [reply]
sudo ./cmd_linux.sh  
 python webui.py

**Comment by grayfox777x** [comment]
I think it just died for me after doing a bunch of updates. It was working fine for me the other night. maybe my model files corrupted. I just keep getting "Could not find the quantized model in .pt or .safetensors format, exiting..."
